applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -252,6 +242,14 @@ static void stream_qos_set_cb(struct bt_audio_stream *stream)
 		if (ret) {
 			LOG_ERR("Unable to enable stream: %d", ret);
 		}
+
```

Comment:
playing_state should be initated to false, and set to true here if ble_mcs_play_pause() succeeds

Reply:
And I dont think it is necessary with the if (playing_state) here

Reply:
The reason it is initialized to true is because we want the stream to start once the first headset is connected. If we remove the 'if(playing_state)' we risk starting a stream if the gateway is paused and one headset reconnects.